docs/source/workbench/plotwindow.rst

```diff
@@ -27,10 +27,20 @@ Plot Window Context Menu
 ------------------------
 
 .. image:: ../images/Workbench/PlotWindow/PlotWindowContextMenu.png
-    :scale: 70%
+    :height: 400px
+
+Right-clicking within your plot will open the plot context menu.
+From this menu you can:
+- set the axes' scales to linear or logarithmic.
```

Comment:
The formatting here doesn't look quite right here as this is not formatted as a list. I think you just need a blank line between the line first line with the `-` and the previous line.